application-load-balancer/locals.tf

```diff
@@ -42,4 +42,7 @@ locals {
   domain_list       = lookup(var.config, "cdn_domains_list", null) != null ? join(",", keys(var.config.cdn_domains_list)) : ""
 
   config_with_defaults = { slack_alert_channel_alb_secret_rotation = coalesce(try(var.config.slack_alert_channel_alb_secret_rotation, null), "C31KW7NLE") } # Slack ID for P2 alerts channel
+
+  # Does the environment have a CDN configured
+  cdn_enabled = length(try({ for k, v in var.config.cdn_domains_list : k => v if !contains(v, "disable_cdn") }, {})) > 0
```

Comment:
To save my current, and other people's future cognitive load, can we use something more meaningful than `k` and `v`?

Reply:
They are meaningless variable names. Not helpful.

https://github.com/uktrade/terraform-platform-modules/pull/331#discussion_r1950630057